### PR TITLE
File Added : Bootstrap Assignment 5&6

### PR DESCRIPTION
**Pull request by roshnidurugwar**

### CODE REVIEW

**Review by Oshin-250197 — APPROVED** [review]

**Review by arshad05786 — APPROVED** [review]

File: Roshni/Bootstrap/Assignment6/Academic.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                <div class="logo">Job Search</div>
+                <div class="menu">
+                    <a href="Home.html">Home</a>
+                    <a href="Personal.html">Personal</a>
+                    <a href="Academic.html">Academics </a>
+                    <a href="Login.html">Login</a>
+                </div>
+            </nav>
+        </header>
+
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Academic.html]: remove extra line

File: Roshni/Bootstrap/Assignment6/Academic.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                </div>
+            </nav>
+        </header>
+       
+        <h2>Academics Details</h2>
+        <br>
+        <div class="form-group">
+            <label for="pwd">Education</label>
+            <input type="password" class="form-control" id="pwd" placeholder="Education" name="pwd" style="width: 150px;">
+          </div>
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Academic.html]: make correct indentation

File: Roshni/Bootstrap/Assignment6/Academic.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                  </div>
+              </div>
+            </form>
+        <br />
+        <br />
+        <br />
+        <input type="submit" value="submit" name="submit" id="submit" />
+        
+    </body>
+</html>
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Academic.html]: make correct indentation

File: Roshni/Bootstrap/Assignment6/Login.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Login.html]: make correct indentation

File: Roshni/Bootstrap/Assignment6/Login.html [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    </head>
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Login.html]: remove this closing tag

File: Roshni/Bootstrap/Assignment6/Login.html [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+                    <a href="Login.html">Login</a>
+                </div>
+            </nav>
+        </header>
+        <h2>Login Form</h2>
+        <form>
+          <div class="imgcontainer">
+            <img src="home.webp" alt="Avatar" class="avatar" style="width: 100px;height: 100px;"> 
+          </div>
+        <div class="container">
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Login.html]: correct indentation

File: Roshni/Bootstrap/Assignment6/Login.html [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+              <input type="checkbox" checked="checked" name="remember"> Remember me
+        </label>
+          </div>
+        <div class="container" style="background-color:#f1f1f1">
+            <button type="button" class="cancelbtn">Cancel</button>
+            <span class="psw">Forgot Password<a href="#"> Forgot password?</a></span>
+        </div>
+    </form>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Roshni/Bootstrap/Assignment6/Login.html:165]: make correct indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Roshni/Bootstrap/Assignment6/Academic.html (hunk 1 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Academic.html]: done

File: Roshni/Bootstrap/Assignment6/Academic.html (hunk 2 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Academic.html]: done

File: Roshni/Bootstrap/Assignment6/Academic.html (hunk 3 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Academic.html]: done

File: Roshni/Bootstrap/Assignment6/Login.html (hunk 4 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Login.html]: done

File: Roshni/Bootstrap/Assignment6/Login.html (hunk 5 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Login.html]: done

File: Roshni/Bootstrap/Assignment6/Login.html (hunk 6 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Login.html]: done

File: Roshni/Bootstrap/Assignment6/Login.html (hunk 7 above)
Comment (roshnidurugwar) [reply Roshni/Bootstrap/Assignment6/Login.html:165]: done

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**